tdrs-backend/tdpservice/templates/error_pages/500.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<!-- saved from url=(0042)https://tdp-frontend-raft.app.cloud.gov/lk -->
```

Comment:
@raftmsohani should this URL be dynamic? i noticed that the TANF Data Portal link on this page redirects to raft's frontend app.

Reply:
@ADPennington this is just a comment, but you are correct, the link should be dynamic.